projects/api-client/src/api/artifact.service.ts

```diff
@@ -2,7 +2,7 @@
  * Wharf main API
  * Wharf backend API that manages data storage for projects, builds, providers, etc.
  *
- * OpenAPI spec version: v4.1.0
+ * OpenAPI spec version: local docker-compose
```

Comment:
Should only building using the latest *released* version of the APIs

Suggest reverting this commit. Why did you need it regenerated anyway?

Reply:
Classifying this as an oops from having worked on the test results display.
I will regenerate with the proper versions, to make sure it's up-to-date.

Reply:
I think regenerating these are out of scope for this PR though

Reply:
I see.
Will just revert then.